dags/public-health/ratstat.py

```diff
@@ -0,0 +1,64 @@
+import arcgis 
+import os 
+from sqlalchemy import create_engine
+from airflow import DAG
+from airflow.operators.python_operator import PythonOperator
+from airflow.hooks.base_hook import base_hook
+from datetime import datetime, timedelta
+import logging
+pg_conn = BaseHook.get_connection('postgres_default') 
+
+def ratstat_loader():
```

Comment:
@hunterowens can you comment here, since you started this DAG?

Reply:
Sorry - I didn't intend to change the ratstat.py or the requirements.txt files in this commit. Not quite sure how that happened with this push.

Reply:
I think you may have branched off of #88 rather than master.